lib/challenge.js

```diff
@@ -48,7 +57,13 @@ var pet_owner;
   // Every pet can:
   // - make noise (each pet makes its own unique noise, as specified by `noise`.
 
-var some_pet;
+var some_pet = {
+  name:'Baxter',
+  species: 'dog',
+  breed:'Golden Retriver'
+  noise: 'bark',
+  makeNoise:
+}
```

Comment:
Aside from not having the requested method and the missing comma, this is a well-formed object literal. Half credit.